pkg/cmd/root/help_topic.go

```diff
@@ -73,6 +73,19 @@ var HelpTopics = map[string]map[string]string{
 			- %[1]stimeago <time>%[1]s: renders a timestamp as relative to now
 			- %[1]spluck <field> <list>%[1]s: collects values of a field from all items in the input
 			- %[1]sjoin <sep> <list>%[1]s: joins values in the list using a separator
+			- %[1]struncate <length> <input>%[1]s: truncates input longer than length and adds "..."
+			- %[1]stable: starts a table. optional arguments match text/tabwriter.NewWriter after first argument
+			- %[1]srow <fields>: writes fields using elastic tabstops
```

Comment:
The user is not aware that tab characters were ever used or that Elastic Tabstops was the underlying algorithm for rendering a table. This is an implementation detail, so we don't need to mention that in the docs